x-pack/platform/plugins/shared/aiops/public/embeddables/log_rate_analysis/log_rate_analysis_embeddable_initializer.tsx

```diff
@@ -141,7 +141,7 @@ export const LogRateAnalysisEmbeddableInitializer: FC<
         }}
       >
         <EuiTitle size="s" data-test-subj="inlineEditingFlyoutLabel">
-          <h2>
+          <h2 id="logRateAnalysisConfig">
```

Comment:
We have an aria-labelledBy defined, but no element had this id. I added it here.